tests/test_logic.py

```diff
@@ -69,7 +69,7 @@ def process(self, instance):
 
 def test_register_gui():
     """Registering at run-time takes precedence over those from environment"""
-    
+
     with no_guis():
         os.environ["PYBLISHGUI"] = "second,third"
```

Comment:
> I have never actually used PYBLISHGUI. When was this introduced?

If you browse to it in the code, and click the `Blame` button, it'll show you when.

> Should this be PYBLISH_GUI instead?

Yes, indeed it should. If you jump on this, don't forget to keep the old one in place as well, for backwards compatibility. We'll document the new one, and then one day in the future we can deprecated it.